include/seqan3/alignment/pairwise/policy/banded_score_dp_matrix_policy.hpp

```diff
@@ -89,9 +94,16 @@ class banded_score_dp_matrix_policy :
         band_row_index = std::abs(std::min(static_cast<int_fast32_t>(band.lower_bound),
                                            static_cast<int_fast32_t>(0)));
 
+        band_size = band_column_index + band_row_index + 1;
+
         // Reserve one more cell to deal with last cell in the banded column which needs only the diagonal and up cell.
         // TODO: introduce specific named cell types with initialisation values.
-        score_matrix.resize(band_column_index + band_row_index + 2, cell_type{INF, INF});
+        score_matrix.resize(band_size + 1);
+
+        using std::get;
+        get<0>(score_matrix.back()) = INF;
```

Comment:
No it is actually the lowest() / 2. It is some detail used inside the algorithm to compute the last value of a column inside a band. It is necessary to avoid underflows in the kernel and to avoid unnecessary instructions.